src/us/kbase/common/executionengine/SubsequentCallRunner.java

```diff
@@ -141,6 +154,27 @@ public ModuleRunVersion getModuleRunVersion() {
             return jsonRpcResponse;
         }
     }
+    
+    private void logError(String name, Integer code, String message, String data) {
+        cbLogErr("Error is thrown by subjod");
```

Comment:
mispelled subjob, I'd probably say `<moduleName>.<methodName> job <jobid> threw an error:` or something like that.